lib/Runtime/Base/FunctionBody.cpp

```diff
@@ -4349,6 +4349,14 @@ namespace Js
         this->RecordConstant(location, str);
     }
 
+    void FunctionBody::RecordBigIntConstant(RegSlot location, LPCOLESTR psz, uint32 cch, bool isNegative)
+    {
+        ScriptContext *scriptContext = this->GetScriptContext();
+        JavascriptBigInt *result = RecyclerNew(scriptContext->GetRecycler(), JavascriptBigInt, scriptContext->GetLibrary()->GetBigIntTypeStatic());
+        Var bigintConst = JavascriptBigInt::Initialize(psz, cch, isNegative, scriptContext, result);
```

Comment:
I still don't understand why we have construction and initialization as two separate steps. You can have multiple constructors, and you can pass any number of arguments to RecyclerNew which get forwarded to the constructor.

Reply:
sorry I missed this one. will do.